### ISSUE TITLE
Connect may have been retired, but the open bugs and issues still exist

### ISSUE THREAD

**Issue by StingyJack** [MicrosoftDocs/MicrosoftCollaboratePortal#67]
And now I cant find things I have logged or bookmarked before. _Useful_ workarounds and such are gone, not to mention **Open** issues that hadn't been resolved. 

Bookmarks for pages like this no longer work and instead go to the "Sorry, connect has been retired" page.

https://connect.microsoft.com/VisualStudio/feedback/details/667994/sqlcmd-execution-r-not-supported-correctly

https://connect.microsoft.com/SQLServer/feedback/details/3143400

These links need to work and take me to the new place where these items are located.

**Comment by ikhapova** [comment]
Thank you for you feedback. Both Visual Studio and SQL server are not working with Collaborate for their customer communities.

**Comment by StingyJack** [comment]
@ikhapova - don't microfocus on the specifics of the links involved. I cited two links that no longer work. but there are many more in other programs as well. 

For many years we went to Connect to get help and many times got that help. The valuable information in the issues reported there is still relevant, but is all is no longer reachable. Some of it was my own contribution or issues that I had opened.. All gone, as if it never happened.
